front/public/locales/de/operationalStudies/home.json

```
@@ -0,0 +1,24 @@
+{
+  "confirmDeleteMessage": "Möchten Sie das Projekt wirklich löschen?",
+  "confirmDeleteMessage_one": "Möchten Sie 1 Projekt wirklich löschen?",
+  "confirmDeleteMessage_other": "Möchten Sie {{ count }} Projekte wirklich löschen?",
+  "createProject": "Projekt anlegen",
+  "deleteItems": "Löschen",
+  "filterPlaceholder": "Suchen...",
+  "openProject": "Öffnen",
+  "projects": "Projekte",
+  "projectsCount_one": "1 Projekt",
+  "projectsCount_other": "{{count}} Projekte",
+  "projectsCount_zero": "Kein Projekt",
+  "safeWordActivated": "„Schlüsselwort“ aktiviert",
```

```suggestion
  "safeWordActivated": "Sicherer Modus aktiviert",
```

to stick to [english](https://github.com/OpenRailAssociation/osrd/blob/dev/front/public/locales/en/operationalStudies/home.json#L14) translation